Demo/Demo/SwiftUIComponents/CardVaultViews/CreateOrderVaultView.swift

```diff
@@ -0,0 +1,66 @@
+import SwiftUI
+
+struct CreateOrderVaultView: View {
+
+    let selectedMerchantIntegration: MerchantIntegration
+    @Binding var intent: String
```

Comment:
Do we need both an intent Binding and selectedIntent State? These seem to do the same thing.

Reply:
the selectIntent state is used for toggling values in the CreateOrderVaultView but the intent as a binding var is passed up to parent view so that it can be used in capture/authorize

Reply:
It was for a silly reason that I wanted to communicate this Intent change to parent view and there was already an existing Intent that is used in UserDefaults in BaseViewModel for CardPayment module. I think it'd be easier for me to make these changes in the refactor PR.

Reply:
Shouldn't the `intent` value live in the ViewModel layer anyways? This way we could use a `@Published` refernce the binding for intent with `$`